packages/overlay/README.md

```diff
@@ -95,6 +96,8 @@ type OverlayOptions = {
 
 `receivesFocus` tells the overlay stack to throw focus into the overlay after it has opened.
 
+`notImmediatelyClosable` prevents immediate clicks from closing the overlay. For example, since `focus` events are dispatched before `click` events, set `notImmediatelyClosable: true` when triggering an overlay via `focus`.
```

Comment:
Not sure if there's a bit of nuance here missing. "hover" content should trigger on `focus`, but be cancelled on `click`, so it doesn't use `notImmediatelyClosable`, whereas "click" or "modal" content that should open on `focus` do, because they _shouldn't_ be closed on the ensuing `click` event. I may be over thinking it.

Reply:
That's good nuance to capture. What do you think of the latest description?